tests/eeid_plugin/plugin/eeid_verifier.c

```diff
@@ -0,0 +1,144 @@
+
```

Comment:
I believe this file should be in the same place with https://github.com/openenclave/openenclave/blob/aa1ebcb5a9d8db12a95932d20fa56f7159395d9a/common/sgx/verifier.c, but with the experimental tag. The users of EEID plugin shouldn't need to rewrite the functionalities here.

Reply:
That's right; I think the whole plugin should move out of the test directory. Would `enclave/sgx` be the right for the EEID attester, next to the SGX attester?

Reply:
Maybe `openenclave/common` is a better place if we want to extend the concept of EEID to TrustZone. @dthaler ?

Reply:
Only files that are truly SGX-specific should be under an sgx directory.   Files that are intended to be agnostic should not be (and should not have SGX-only code in such files).